app/views/shared/_print_footer.html.erb

```diff
@@ -1,13 +1,13 @@
 <footer class="<%= (last ? 'sticky_footer footer' : 'footer--print footer') %>">
   <div class="copyright">
     <% if @footer && @footer.first_part %>
-      <%= image_tag "https://i.creativecommons.org/l/by-nc-sa/3.0/88x31.png", class: 
+      <%= image_tag "https://i.creativecommons.org/l/by-nc-sa/3.0/88x31.png", class:
```

Comment:
I think we're going to want https://licensebuttons.net/l/by-nc-sa/4.0/88x31.png here to display the correct license badge for CC-by-SA 4.0.

Reply:
Also, do we need a line break between `class:` and the class name?

Reply:
fixed.